drivers/afe/ad4110/ad4110.c

```diff
@@ -308,14 +308,14 @@ int32_t ad4110_set_op_mode(struct ad4110_dev *dev, enum ad4110_op_mode mode)
 		return ad4110_spi_int_reg_write_msk(dev,
 						    A4110_AFE,
 						    AD4110_REG_AFE_CNTRL2,
-						    AD4110_ENABLE,
+						    AD4110_REG_AFE_CNTRL2_IMODE_MSK,
```

Comment:
we shouldn't pass masks as data parameters.
Please use `no_os_field_prep` utility function inline:
`no_os_field_prep(AD4110_REG_AFE_CNTRL2_IMODE_MSK, AD4110_ENABLE)`.

Or create similar defines to: https://github.com/analogdevicesinc/no-OS/blob/4dc973f8be0ccd623e4b1e5e2a11b5bc33d0b944/drivers/afe/ad4110/ad4110.h#L123-L124

Reply:
The change has been made- no_os_field_prep() has been used